### PR TITLE
update s2i version

### PR DESCRIPTION
**Pull request by runzexia**

### CODE REVIEW

**Comment by runzexia** [comment]
@zryfish this PR need to be merge

File: go.mod [hunk 1]
```diff
@@ -78,7 +78,7 @@ require (
 	github.com/mholt/certmagic v0.5.1 // indirect
 	github.com/miekg/dns v1.1.9 // indirect
 	github.com/mitchellh/go-homedir v1.1.0 // indirect
-	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect; indirec
```
Comment (zryfish) [inline go.mod]: Is this a typo?

Comment (runzexia) [reply go.mod]: `go mod tidy` generated it

Comment (runzexia) [reply go.mod]: fix

Comment (zryfish) [reply go.mod]: I can see it's still there

**Pull request merged**